localstack/services/cloudformation/service_models.py

```diff
@@ -39,6 +39,7 @@ class GenericBaseModel(CloudFormationModel):
     """
 
     def __init__(self, resource_json, region_name=None, **params):
+        # self.stack_name = stack_name # TODO: add stack name to params
```

Comment:
That's a great change to make in the future - will simplify things quite a bit. 👍